manifest.json

```
@@ -22,7 +22,7 @@
   "permissions": ["storage", "*://content.guardianapis.com/*"],
   "browser_specific_settings": {
     "gecko": {
-      "id": "sindicator@theguardian.com",
+      "id": "sindicator@the-guardian.com",
```

I’d be a little worried about using a domain we don’t own for this! Could we change it to guardian.co.uk?

```suggestion
      "id": "sindicator@guardian.co.uk",
```

…or change the name, maybe by including the uppercase I:

```suggestion
      "id": "sIndicator@theguardian.com",
```

---

To be clear, I don’t know what the impact of using a domain we don’t own is: maybe it’d be fine! But I worry that it’ll give the owner of that domain some control or ownership when interacting with the firefox extension store, for example.

---

It's not case sensitive - so I've gone for sindicator@guardian.co.uk

---

I accidentally left a comment pending earlier that said:

> To be clear, I don’t know what the impact of using a domain we don’t own is: maybe it’d be fine! But I worry that it’ll give the owner of that domain some control or ownership when interacting with the firefox extension store, for example.